docs/how-to-guides/meson-args.rst

```diff
@@ -44,6 +44,16 @@ building Python wheel. User options specified via ``pyproject.toml``
 or via Python build front-end config settings can be used to override
 the ``meson-python`` defaults..
 
+When building on Windows, ``meson-python`` invokes the ``ninja``
+command via the ``meson compile`` wrapper. This activates the Visual
+Studio environment and allows ``ninja`` to use the MSVC compilers. The
```

Comment:
This is not necessarily true, it depends on what compiler meson selected at setup time (which depends compilers you have in the PATH.).
```suggestion
Studio environment (if that was the compiler meson selected at setup time) and allows ``ninja`` to find the MSVC compilers. The
```

Reply:
Thank for the review. I was trying to keep the explanation as simple as possible. I really wanted to refer to the Meson documentation for the details of the compilers discovery, but I haven't found the section of the Meson manual where it is described. I thought hat "allows ninja to use the MSVC compilers" didn't imply that these are not necessarily the ones that will be used. Maybe adding more detail is required. Let me try again.

Reply:
I've extended the documentation. Please take a look.